src/cwv/kpi-metrics.js

```diff
@@ -72,17 +74,24 @@ const calculateProjectedTrafficLost = (metrics) => {
   return metrics.organic * trafficMultiplier;
 };
 
-const calculateKpiDeltasForAuditEntryPerDevice = (entry) => {
+const calculateProjectedTrafficValue = (
+  projectedTrafficLost,
+  cpcValue,
+) => projectedTrafficLost * cpcValue;
+
+const calculateKpiDeltasForAuditEntryPerDevice = (entry, cpcValue) => {
```

Comment:
You can use `reduce` instead of `forEach` when building objects to avoid mutating the object directly and keeps the things more javascript-ish functional:

```js
const calculateKpiDeltasForAuditEntryPerDevice = (entry, cpcValue) => {
  return entry.metrics.reduce((kpiDeltas, metrics) => {
    const { deviceType } = metrics;

    const projectedTrafficLost = calculateProjectedTrafficLost(metrics);
    const projectedTrafficValue = calculateProjectedTrafficValue(projectedTrafficLost, cpcValue);

    return {
      ...kpiDeltas,
      [deviceType]: {
        projectedTrafficLost,
        projectedTrafficValue,
      },
    };
  }, {});
};
```